### ISSUE TITLE
Improve error in Safari private mode

### ISSUE THREAD

**Issue by rozbb** [rozbb/readtomyshoe#24]
IndexedDB is not available in Safari's private mode, so RTMS fundamentally does not work. Make it error on add-to-queue, with error text nicer than attached.
<img width="883" alt="Screen Shot 2022-09-12 at 00 13 17" alt="RTMS homepage with library section containing red text, reading out an error that's pages long" src="https://user-images.githubusercontent.com/752802/189572623-e8f04da5-0eab-44e3-b1f3-f8c4f7fa356b.png">
